project/build.scala

```diff
@@ -143,8 +112,43 @@ object Shared {
     resolvers += Resolver.sonatypeRepo("releases"),
     shellPrompt := ShellPrompt.buildShellPrompt,
     organization := "org.capnproto",
-    publishArtifact := false
+    publishArtifact := false,
+    publishTo := {
+      val nexus = "https://oss.sonatype.org/"
+      if (isSnapshot.value)
+        Some("snapshots" at nexus + "content/repositories/snapshots")
+      else
+        Some("releases"  at nexus + "service/local/staging/deploy/maven2")
+    },
+    credentials += Credentials(Path.userHome / ".ivy2" / ".credentials"),
+    publishMavenStyle := true,
+    version := "0.1.0-SNAPSHOT",
+    publishArtifact in Test := false,
+    pomIncludeRepository := { x => false },
+    pomExtra := (
+      <url>https://capnproto.org/</url>
```

Comment:
I mean in the pom of the runtime-0.1.0-SNAPSHOT you see the scala library dependency even though there is no code dependency, sbt will auto add this to pom unless we put autoScalaLibrary := false